apps/poker-roll/src/index.html

```diff
@@ -1,14 +1,18 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <title>PokerRoll</title>
-    <base href="/" />
 
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="icon" type="image/x-icon" href="favicon.ico" />
-  </head>
-  <body>
-    <div id="root"></div>
-  </body>
-</html>
+<head>
+  <meta charset="utf-8" />
+  <title>PokerRoll</title>
+  <base href="/" />
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
+    integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous" />
```

Comment:
I'd recommend learning another way to do styling instead of falling back to Bootstrap. The goal is to keep exposing yourself to new tech so when you come across it or something similar again later you're already a little bit familiar with how it works. Also good to keep getting practice working through documentation.

Material UI's `sx` API is fairly similar to Bootstrap's classes. For example, I see you're using "bg-white m-4" on your form for a white background with a margin of 4. In Material that translates to `sx={{ m: 4, bgColor: white }}`. VSCode IntelliSense will even show you the autocomplete options when you start adding styles within `sx`.

Reply:
yes, good advice. i removed the bootstrap styles